src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.module.scss

```diff
@@ -19,11 +19,11 @@
 }
 
 .desktopArticleVideoWrapperScrollPlayer {
-	height: max-content;
+  	height: 168.75px;
```

Comment:
suggestion: I'd say we should check if it works the same with max-content, if yes, let's use it, if not, let's round it to 169px as 1px is the smallest unit

Reply:
It doesn't work properly with max-content. 168.75px is the exact height of the floated player